### DISCUSSION TITLE
Antd v6?👀

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#48750] [Q&A]
Hi guys, any release date for antd v6?
What will be the changelog🙂?

**Comment by shivarajnaidu** [comment]
Any plans or discussions links related to v6?

**Comment by HaceraI** [comment]
Yes, it is currently in progress


discussions: https://github.com/ant-design/ant-design/discussions/51919
task: https://github.com/ant-design/ant-design/issues/52115

**Comment by prakashmallow** [comment]
@HaceraI @zombieJ @meet-student  When can we expect the release of Ant Design v6? We’re currently facing several accessibility issues with the existing components and would like to know if these issues are expected to be addressed in version 6.

**Reply by meet-student** [reply]
It is planned to be released on November 22nd, China time.
accessibility issues related to on-demand fixes can be raised.

v6 links

discussions: https://github.com/ant-design/ant-design/discussions/51919
task: https://github.com/ant-design/ant-design/issues/52115

**Comment by roflc0pter** [comment]
Already started using. No issues so far. Thanks for the great job!
Neither the change log nor the discussion thread or the documentation says why the List component got deprecated. What's the actual reason to kill the List component in the future?

**Reply by meet-student** [reply]
- #54159
- #54458
